content/analyze/methods/methods.md

```diff
@@ -9,3 +9,13 @@ linked:
     - ./scmap.md
     - ./stream.md
 ---
+
+##  Methods Packages
+The HCA Methods Package Registry lists tools for performing analyses involving scRNA-seq, scDNA-seq, in situ transcriptomics, multi-modal and other computational biology approaches. 
+
+These solutions are not built by the HCA DCP implementation team but are solutions we foster and support. 
```

Comment:
A bit shorter and less redundant, while preserving Tim's original intent to distinguish:

```suggestion
These solutions are built by third parties, and fostered by the HCA DCP.
```